platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -214,13 +493,7 @@ message RosterState {
      */
     repeated RoundRosterPair round_roster_pairs = 2;
```

Comment:
Since the proposal mostly mentions this list as `rosterHistory`, we should consider renaming the `round_roster_pairs` to `roster_history`

Reply:
Is it not too late in the cycle to make these field name changes?    I would prefer it be called `rosterHistory` as well.